index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Apple_Page</title>
+    <!-- Icons -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" 
+    crossorigin="anonymous">
+    <!-- css -->
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="hero">
+    <!-- nav start -->
+    <main>
+    <nav class="nav-header">
+            <ul class="nav-list">
+                <li><a href="#"><i class="fab fa-apple icon-apple"></i></a></li>
+                <li class="font-size"><a href="#">Store</a></li>
+                <li class="font-size"><a href="#">Mac</a></li>
+                <li class="font-size"><a href="#">iPod</a></li>
+                <li class="font-size"><a href="#">iPhone</a></li>
+                <li class="font-size"><a href="#">iPad</a></li>
+                <li class="font-size"><a href="#">iTunes</a></li>
+                <li class="font-size"><a href="#">Support</a></li>
+                <li>
+                    <div class="nav-input">
+                        <form action="#">
+                            <i class="fas fa-search icon"></i>
+                            <input class="input-field" type="search" size="10">
+                        </form>
+                    </div>
+                </li>
+            </ul>
+        </nav>
+        <!-- nav end -->
+    </main>
+        <!-- Hero section -->
+            <section>
+            <!-- Hero Header -->
+                <div class="hero-header">
+                    <h1 class="one"> 
+                        <a href="#">
+                            <img src="images/what.png" alt="what" width="980" height="104">
+                        </a>
+                    </h1>
+                    <h2 class="two">
+                        <a href="#">
+                            <img src="images/everyone.png" alt="everyone" width="980" height="82">
+                        </a>
+                    </h2>
+                    <div>
+                        <a href="#">
+                            <img src="images/play-story.png" alt="Play" width="980" height="65"> 
+                        </a>
+                    </div>
+                </div>
+            </section>
+    </div>
+
+            <!-- Promos -->
+    <div class="promo">
```

Comment:
You can use a `section` tag here instead of `div`. HTML5 have a lot of new tags that can be used to avoid using div tags. Div tags have no semantic value, so its best practice to not use it when you can use another tag instead. You can learn more about semantic HTML [here](https://internetingishard.com/html-and-css/semantic-html/)